containertest/docker_test.go

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package mysqltest
+package containertest
 
 // This file is only intended to be used outside of Google. Inside of Google, this file should be
 // replaced with the Google-internal version.
 
 import (
 	"database/sql"
 	"fmt"
+	"net"
 	"strings"
 	"testing"
 	"time"
 )
 
+// MySQL will be used as the default service for testing general behavior. Other
+// services should have their own tests written.
 func TestKillAfter(t *testing.T) {
```

Comment:
This is flaky on my box :(

Reply:
I extended startup time to 2 minutes but still sometimes it doesn't start up in time.

Reply:
I fixed it by forcing 5.7, it seems the latest image (docker 8) is taking a lot longer to boot sometimes.